build_catalog.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+num_commits=$(git rev-list $(git rev-list --max-parents=0 HEAD)..HEAD --count)
+current_commit=$(git rev-parse --short=7 HEAD)
+version="0.1.$num_commits-$current_commit"
+opm_version="1.14.0"
+
+# Login to docker
+docker_conf="$PWD/.docker"
+mkdir -p "$docker_conf"
+docker_cmd="docker --config=$docker_conf"
+
+$docker_cmd login -u="$QUAY_USER" -p="$QUAY_TOKEN" quay.io
+
+# Find the CSV version from the previous bundle
+$docker_cmd pull $BUNDLE_IMAGE_NAME:latest && exists=1 || exists=0
```

Comment:
if you're depending from external variables you should check for them to exist or at least add `set -o no unset`

Reply:
Checking for the env names.  Can't use -o nounset because there is a variable that will be unbound in some cases (previous version)